ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -111,7 +111,7 @@ const ServiceDetailsScreen = (props: Props) => {
 
   useEffect(() => {
     props.loadServiceDetail(props.serviceId);
-  }, []);
+  }, [props]);
```

Comment:
Why you want to depend on whole props?

What you think about?

```
  const { loadServiceDetail, serviceId } = props;

  useEffect(() => {
    loadServiceDetail(serviceId);
  }, [loadServiceDetail, serviceId]);
```

Reply:
Yep, you'll find this on #4070